frdrpc/rpc.proto

```diff
@@ -294,3 +306,37 @@ message ChannelInsight {
     // True if the channel is private.
     bool private = 8;
 }
+
+message FiatEstimateRequest {
+    message PriceRequest {
+        string id = 1;
+
+        // The amount to be converted to fiat in millisatoshis.
+        uint64 amount_msat = 2;
+
+        // The unix timestamp at which the price should be quoted.
+        int64 timestamp = 3;
+    }
+
+    repeated PriceRequest requests = 1;
```

Comment:
Is there a specific reason to allow multiple requests to be sent at a time? The API is not really intuitive that way with the request ID and the map as a response. But maybe it's needed for a follow-up PR?

Reply:
This functionality is mostly needed to produce fiat quotes en-masse for an accounting spreadsheet. So it'll be much more efficient to make one large query. 

However, that's going to be done internally in faraday, so rpc could be simplified to just be a single request like the cli? Really just added it for easy testing/ because we might as well have the feature.

Reply:
It's okay leaving it that way, was mostly curious. Could be useful for other internal projects to batch multiple requests together.